HW5/HW5 SQL.sql

```diff
@@ -0,0 +1,126 @@
+CREATE OR REPLACE TABLE temp_for_rolling 
+SELECT 
+	g.game_id AS game_id,
+	tbc.team_id AS team_id,
+	g.local_date AS game_date,
+	tbc.homeTeam AS is_home,
+	tbc.finalScore AS final_score,
+	tbc.atBat AS off_at_bat,
+	tbc.Hit AS off_hit,
+	tbc.plateApperance AS off_plate_appr,
+	tbc.Strikeout AS off_strikeout,
+	tbc.Single as off_single,
+	tbc.Double as off_double,
+	tbc.Triple as off_triple,
+	tbc.Walk as off_walk,
+	tbc.Ground_Out + tbc.Groundout as off_ground,
+	tbc.Fly_Out + tbc.Flyout as off_flyout,
+	tpc.atBat AS def_at_bat,
+	tpc.Hit AS def_hit,
+	tpc.plateApperance AS def_plate_appr,
+	tpc.Strikeout AS def_strikeout,
+	tpc.Single as def_single,
+	tpc.Double as def_double,
+	tpc.Triple as def_triple,
+	tbc.Walk as def_walk,
+	tpc.Ground_Out + tpc.Groundout as def_ground,
+	tpc.Fly_Out + tpc.Flyout as def_flyout
+	
```

Comment:
I thought the features for `off_` and `def_` are when the `homeTeam` value is 0 or 1, meaning awayteam or hometeam?

Reply:
Off is offensive and Def is defensive. For example, off_batting_avg is the average when the team is an offensive team.